plugin-developer-learning-pathway/07-Custom-Database-Tables/01-Creating-and-Managing-Custom-Tables.md

```diff
@@ -16,54 +18,76 @@ Even still, there are times when creating a custom table can be more effective.
 
-The `wpdb` class is used to manage a database connection while also providing helpful methods to perform queries and retrieve data from the connected database in a structured way. In PHP, WordPress automatically sets the `$wpdb` global variable to an instance of this class by using the `DB_HOST`, `DB_NAME`, `DB_USER`, and `DB_PASSWORD` constants defined in your `wp-config.php` file.
+During PHP execution, WordPress creates a `$wpdb` global object variable as an instance of this class by using the `DB_HOST`, `DB_NAME`, `DB_USER`, and `DB_PASSWORD` constants defined in your `wp-config.php` file.
 
-You can access it in PHP like this:
+Because the `$wpdb` object exists in the global namespace, you can access it in PHP like this:
 
 ```php
 global $wpdb;
 ```
 
-The `$wpdb` instance also contains crucial information about the WordPress database which we'll need to properly create and manage our own custom tables. Let's see how as we create our first one.
+The `$wpdb` instance also contains crucial information about the WordPress database which you'll need to properly create and manage our own custom tables. Let's see how as we create our first one.
 
 ## Creating a Custom Table
 
-To start, register an activation hook for your plugin to create the custom table. Using this hook ensures the table immediately exists once your plugin is activated so that any following code can safely query against it.
+To start, create a new plugin to store the custom table code. 
+
+```
+<?php
+/**
+ * Plugin Name: Form submissions
+ * Version:     1.0.0
+ * Author:      Learn WordPress
+ * Author URI:  https: //learn.wordpress.org/
+ *
+ * @package wp-learn-form-submissions
+ */
+ ```
+
+Then, register an activation hook for your plugin to create the custom table. Using this hook ensures the table immediately exists once your plugin is activated so that any following code can safely query against it.
 
-When choosing a name for the custom table, be sure to use the configured table prefix by prepending the table name with the `$wpdb->prefix` variable. Also, use `$wpdb->get_charset_collate()` to ensure the custom table's collation matches the rest of the tables in WordPress for optimum data integrity, compatibility, and query performance.
+When choosing a name for the custom table, be sure to use the configured table prefix by prepending the table name with the `$wpdb->prefix` object property. Also, use `$wpdb->get_charset_collate()` method to ensure the custom table's collation matches the rest of the tables in WordPress for optimum data integrity, compatibility, and query performance.
 
-To create a custom table, use the `dbDelta()` function included in WordPress. This function is powerful because it can handle both table creation and future updates to the table's structure.
+To create a custom table, use the `dbDelta()` function included in WordPress. This function handles both table creation and future updates to the table's structure.
```

Comment:
We said "use" a lot in the previous paragraph, so kinda want to break up the demanding voice that's piling up by softening this command a bit like:

To create a custom table, **its best to** use the `dbDelta()` function...

Reply:
"It's best" makes it sound to me like there's an alternative option, which there really isn't. 

Maybe "To create a custom table, WordPress offers the `dbDelta` function. This function..." sounds better?

Reply:
True, it makes it sound like a suggestion/recommendation rather than simpler stating what ought be used.

When reading it aloud with a different inflection, I think I can make the repetitive usage of "use" work. If I read it like a book/narration, then it sounds repetitive. If I read it more like a bulleted list, then it makes more sense.

With that in mind, I'll mark this as resolved with no change needed.